valhalla/baldr/graphid.h

```diff
@@ -224,7 +224,14 @@ struct GraphId {
 namespace std {
 template <> struct hash<valhalla::baldr::GraphId> {
   inline std::size_t operator()(const valhalla::baldr::GraphId& k) const {
-    return static_cast<size_t>(k.value);
+    // simplified version of murmur3 hash for 64 bit
+    uint64_t v = k.value;
```

Comment:
nit: should we extract it into separate function in order to (maybe) re-use it for other types?

Reply:
There is already similar function `robin_hood::hash_int`, however it is even more simplified and requires `#include <robin_hood.h>` in this header

Reply:
Yeah we don't robinhood in our public headers